pxr/imaging/rprUsd/materialNodes/mtlxFiles/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+set(MTLX_FILES
```

Comment:
could we do Patterns/*.mtlx?  Or do you want to list explicedly.

Reply:
Globbing in cmake is a bad smell. One of the reasons for it is that when you add a file cmake will not update the globbing query. You need to do it explicitly by resetting the cache. So listing your files explicitly, in general, is the better approach